### PR TITLE
Added required Elements to Editor.js

### PR DESCRIPTION
**Pull request by mvskiren**
Input and output Textarea, Run button, Select dropdown

### CODE REVIEW

File: src/Editor/Editor.jsx [hunk 1]
```diff
@@ -3,7 +3,8 @@ import { Controlled as CodeMirror } from "react-codemirror2";
 import "codemirror/lib/codemirror.css";
 import "codemirror/theme/material.css";
 import "codemirror/mode/clike/clike";
-import 'codemirror/keymap/sublime'
```
Comment (dharanad) [inline src/Editor/Editor.jsx:6]: Failed to compile.

./src/Editor/Editor.jsx
Cannot find file './style.module.css' in './src/Editor'.

File: src/Editor/Editor.jsx [hunk 2]
```diff
@@ -34,17 +35,37 @@ class Editor extends Component {
   };
 
   render() {
-    
+
     return (
-      <div style={{width:'75%'}}>
-        <CodeMirror
+      <div style={{width:'75%',margin:'4%'}}>
```
Comment (dharanad) [inline src/Editor/Editor.jsx:40]: These are static option. Can't attach a click listener to each option. Make component which takes in a list and render the list as option with onClickListener attached to them.

**Review by dharanad — COMMENTED** [review]
Need Changes.

**Comment by dharanad** [comment]
I have made changes to Editor Component. If contributing checkout the latest commit and make changes accordingly.

File: src/Editor/Editor.jsx (hunk 2 above)
Comment (mvskiren) [reply src/Editor/Editor.jsx:40]: Didn't get that...

File: src/Editor/Editor.jsx (hunk 1 above)
Comment (mvskiren) [reply src/Editor/Editor.jsx:6]: Ohh I have forgot to commit the CSS file, I will make a pull request now...

File: src/Editor/Editor.jsx (hunk 2 above)
Comment (dharanad) [reply src/Editor/Editor.jsx:40]: You have hardcoded the option in the select tag, instead make a component which takes in a list of string and render that list dynamically. Anyways i have done it here [Selector Component](https://github.com/dharan1011/algo-run/blob/ui/src/Selector/Selector.jsx)

**Pull request closed without merging**